server/api/hooks.js

```diff
@@ -59,6 +59,8 @@ router.post('hooks.slack', async ctx => {
   });
 
   if (!user) throw httpErrors.BadRequest('Invalid user');
+  if (!user.isAdmin)
```

Comment:
We should keep in the back of our mind using cancan like 'policies' or some similar concept in the future to disconnect the users role from hardcoded all over the place, this is totally the right approach for the short term though.

Reply:
I like that. Maybe we'll end up moving `.isAdmin` to be something more granular